src/modules/payment-backend-methods/index.ts

```diff
@@ -14,9 +15,34 @@ const store = {
   }
 }
 
-const KEY = 'payment-backend-methods'
-export const PaymentBackendMethods = createModule({
-  key: KEY,
-  store: { modules: [{ key: KEY, module: store }] },
-  afterRegistration
-})
+export const PaymentBackendMethodsModule: StorefrontModule = function (app, store, router, moduleConfig, appConfig) {
+  store.registerModule('payment-backend-methods', PaymentBackendMethodsStore)
+
+  let correctPaymentMethod = false
+
+  // Place the order. Payload is empty as we don't have any specific info to add for this payment method '{}'
+  const placeOrder = () => {
+    if (correctPaymentMethod) {
+      Vue.prototype.$bus.$emit('checkout-do-placeOrder', {})
```

Comment:
shouldn't we stop using Vue.prototype.$bus? @pkarw did refactor with that and i think just EventBus should be used here

Reply:
this part was only moved but you're right ;)